Platform support for the DELL EMC S4100 with the Rangeley C2338 CPU module

From: Lakesh Sharma <lakesh_sharma@dell.com>


---
 drivers/platform/x86/Kconfig |   14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/drivers/platform/x86/Kconfig b/drivers/platform/x86/Kconfig
index b8a21d7..c19fb92 100644
--- a/drivers/platform/x86/Kconfig
+++ b/drivers/platform/x86/Kconfig
@@ -1027,4 +1027,18 @@ config INTEL_TELEMETRY
 	  used to get various SoC events and parameters
 	  directly via debugfs files. Various tools may use
 	  this interface for SoC state monitoring.
+
+config DELLEMC_S4100_C2338
+	tristate "DELL EMC S4100_C2338 Platform Support"
+	depends on X86 && PCI
+	select I2C_ISCH
+	select I2C_ISMT
+	select GPIO_SCH
+	select MFD_CORE
+	select LPC_SCH
+	---help---
+	Enable support for DELL EMC S4100 switching platform.  This
+	platform has a C2338 Atom (Rangeley) CPU module and Broadcom
+	BCM56760A/BCM56762A as switch ASIC.
+
 endif # X86_PLATFORM_DEVICES
